modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: Procedure
+[id="deploying-book-info"]
+= Deploying the Bookinfo application
+:context: ossm-pdeploying-book-info
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+* You have access to the {ocp-short-name} CLI (oc).
+* You have installed the {SMProductName} Operator, and deployed {istio}.
+* You have created the {istio} and IstioCNI resources.
+
+. In the {ocp-product-title} web console, navigate to the *Home* -> *Projects* page.
+
+. Click *Create Project*.
+
+. Enter `bookinfo` in the *Project name* field.
++
+The *Display name* and *Description* fields provide supplementary information and are not required. 
+
+. Click *Create*.
+
+. Apply the the {Istio} injection label to the `bookinfo` namespace by entering the following command at the command line:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-injection=enabled
+----
+
+. Apply the `bookinfo` YAML file to deploy the `bookinfo` application by entering the following command at the CLI:
++
+[source,terminal]
+----
+oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo
+----
+
+.Verification
+
+. Verify that the `bookinfo` service is available by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc get services -n bookinfo
+----
++
+.Example output
+[source,terminal]
+----
+NAME          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+details       ClusterIP   172.30.137.21   <none>        9080/TCP   44s
+productpage   ClusterIP   172.30.2.246    <none>        9080/TCP   43s
+ratings       ClusterIP   172.30.33.85    <none>        9080/TCP   44s
+reviews       ClusterIP   172.30.175.88   <none>        9080/TCP   44s
+----
+
+. Verify that the `bookinfo` pods are available by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc get pods -n bookinfo
+----
++
+.Example output
+[source,terminal]
+----
+NAME                             READY   STATUS    RESTARTS   AGE
+details-v1-698d88b-km2jg         2/2     Running   0          66s
+productpage-v1-675fc69cf-cvxv9   2/2     Running   0          65s
+ratings-v1-6484c4d9bb-tpx7d      2/2     Running   0          65s
+reviews-v1-5b5d6494f4-wsrwp      2/2     Running   0          65s
+reviews-v2-5b667bcbf8-4lsfd      2/2     Running   0          65s
+reviews-v3-5b9bd44f4-44hr6       2/2     Running   0          65s
+----
++
+Confirm the that `Running` appears in the `Status` column for each pod.
```

Comment:
The pods probably wont be in the `Running` state immediately but should eventually get there so the user should just need to wait until they do. I'm not sure you need to call that out though.

Reply:
You can just ignore this comment. Looks like I can't dismiss them myself.